CHANGELOG.md

```diff
@@ -1,3 +1,191 @@
+# 23.08 (2023-08-31)
+
+A big shout out to all the contributors! We had 107 contributors in this release.
+
+Breaking changes:
+
+- Support multiple language servers per language (#2507)
+    - This is a breaking change to language configuration
+
+Features:
+
+- Support multiple language servers per language (#2507, #7082, #7286)
+- Add a statusline element for the selected register (#7222)
+- Add `%`, `#`, `.`, `*` and `+` special registers (#6985)
+- Add initial support for LSP DidChangeWatchedFiles notifications (#7665)
+- Search buffer contents in `global_search` (#5652)
+- Add a "smart tab" command that intelligently jumps the cursor on tab (#4443)
+- Add a statusline element for whether a file is read-only (#7222, #7875)
+
+Commands:
+
+- `merge_selections` (`A-minus`) - merge all selections into one selection that covers all ranges (#7053)
+- `move_prev_long_word_end` and `extend_prev_long_word_end` - move/extend to the end of the previous WORD (#6905)
+- `reverse_selection_contents` - swaps the values of each register so they are reversed (#7329)
+- Add `:rl` and `:rla` aliases for `:reload` and `:reload-all` (#7158)
+- `yank_joined` - Join the selections and yank to the selected register (#7195)
+- `:write-all!` (`:wa!`) - forcibly write all buffers to disk and create any necessary subdirectories (#7577)
+
+Usability improvements:
+
+- Allow cycling option values at runtime (#4411, #7240, 7877)
+- Exit gracefully on termination signals (#7236)
+- Add plaintext matching fallback to tree-sitter pair matching (#4288)
+- Persist register selection in pending keymaps (0e08349)
+- Propagate the count and register to command palette commands (b394997)
+- Auto indent on `insert_at_line_start` (#5837)
+- Add a config option to control whether LSP completions are automatically inserted on preview (#7189)
+- Check for `git` before fetching or building grammars (#7320)
+- Add a config option for default line endings (#5621, #7357)
+- Allow ANSI colors in themes (#5119)
+- Match pairs that don't form a standalone tree-sitter node (#7242)
+- Allow indent sizes of up to 16 columns (#7429)
+- Improve performance of mapping positions through changes (#7408, 8d39a81, #7471)
+- Mark buffers created from stdin as modified (#7431)
+- Forcibly shut down uninitialized language servers (#7449)
+- Use redraw handle to debounce LSP notifications (#7538, #8023)
```

Comment:
```suggestion
- Fix debouncing of LSP messages to fix the last message sticking around (#7538, #8023)
```

Maybe not ideal yet but I think this needs a less technical description. Nobody but us knows what the redraw handle is :D I think this could also be moved to bug fixes